### DISCUSSION TITLE
Add Qwik integration with Tailwind Elements

### DISCUSSION THREAD

**Discussion by md-owes** [mdbootstrap/TW-Elements#1900] [Share ideas & request features]
Dear author/maintainer. I would love to see official support for Qwik framework which is gaining traction in recent time. Currently using DaisyUI but it has support for far less components.

**Comment by juujisai** [comment]
Hi, thanks for the suggestion. We will keep that in mind.

**Comment by ananta-dev** [comment]
I would love support for Qwik too!

**Comment by RBleyenberg** [comment]
Howdy,

Support native is always nice ^^
Tho, at the moment you can use it without any issues as far as i can tell.

Just take any example code and use the javascript import within the useVisibleTask$

```
    useVisibleTask$(() => {
        initTE({Collapse, Dropdown});
    })
```

And for the install just follow the npm, hope it helps somebody for now.
